perllib/FixMyStreet/DB/Result/User.pm

```diff
@@ -204,6 +204,27 @@ sub alert_for_problem {
     } );
 }
 
+=head2 create_alert
+
+Sign a user up to receive alerts on a given problem
+
+=cut
+
+sub create_alert {
+    my ( $self, $id, %options ) = @_;
```

Comment:
Given every call to this is derefencing a hashref, perhaps accept a hashref instead and deference it inside this function?